Vagrantfile

```diff
@@ -90,11 +90,6 @@ Vagrant.configure('2') do |config|
     config.vm.box = BOX_IMAGE
     config.vm.box_check_update = true
 
-    config.vm.provider 'virtualbox' do |l|
```

Comment:
the master VM is getting set to 2 CPUs and 2048 MB memory below on lines 98-99, so doing it here first is misleading when it will just get overridden later on.